skbio/diversity/alpha/tests/test_base.py

```diff
@@ -393,6 +394,20 @@ def test_observed_otus(self):
     def test_osd(self):
         self.assertEqual(osd(self.counts), (9, 3, 3))
 
+    def test_pielou(self):
+        # Calculate "by hand".
+        arr = np.array([1, 2, 3, 1])
+        h = shannon(arr)
+        s = 4
+        expected = h / np.log(s)
+        self.assertAlmostEqual(pielou(arr), expected)
+
+        self.assertAlmostEqual(pielou(self.counts), 1.334283585856)
```

Comment:
A couple of additional "boundary case" tests would also be good here. Can you test with passing in an empty vector (i.e., ``[]``), a vector where there are counts greater than 1 (e.g., ``[1,2,3,0]``), and a vector where ``observed_otus == 0``? 

Also, minor, but you only need to have the seven most significant digits in your expected values, b/c ``assertAlmostEqual`` only checks for equality to the sixth decimal place by default.